### ISSUE TITLE
Is `-speak` broken?

### ISSUE THREAD

**Issue by admirabilis** [soimort/translate-shell#403]
```
$ trans -speak en: -no-translate Hello
Voice output isn't available for English
$ trans -speak en: -engine google -no-translate Hello
Voice output isn't available for English
$ trans -speak en: -engine bing -no-translate Hello
Voice output isn't available for English
$ trans -speak en: -engine yandex -no-translate Hello
[ERROR] Oops! Something went wrong and I can't translate it for you :(
```

**Comment by soimort** [comment]
Which version (output of `trans -V`), please?

**Comment by admirabilis** [comment]
Oh, sorry: 
```
Translate Shell       0.9.6.12-release

platform              Linux
terminal type         xterm-256color
bi-di emulator        [N/A]
gawk (GNU Awk)        5.0.1
fribidi (GNU FriBidi) [NOT INSTALLED]
audio player          mpv --no-config
terminal pager        less
web browser           xdg-open
user locale           pt_BR.UTF-8 (Portuguese)
home language         pt
source language       auto
target language       pt
translation engine    google
proxy                 [NONE]
user-agent            Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
ip version            [DEFAULT]
theme                 default
init file             [NONE]
```

**Comment by soimort** [comment]
Can you post the output of `trans -V`?

**Comment by admirabilis** [comment]
Done above a few seconds earlier! I installed it with curl.

**Comment by soimort** [comment]
Does it work when specifying an audio player, like `-player mpv`?

**Comment by admirabilis** [comment]
It now works with Google, no difference with the player option, maybe I was being rate limited. With Yandex and Bing, it still doesn't, but I guess that is another issue.

Thanks for your attention!

**Comment by soimort** [comment]
> With Yandex and Bing, it still doesn't, but I guess that is another issue.

Yes, Bing and Yandex supports are known to be broken at the moment.

**Comment by agguser** [comment]
It seems that `-speak` does not work with `-no-translate`; bug?
```
$ trans -speak -no-translate hello   # not speak
Voice output isn't available for English

$ trans -speak -b hello   # speak
hello

$ trans -V
Translate Shell       0.9.7-git:a3bba39
…
```

**Comment by soimort** [comment]
> It seems that `-speak` does not work with `-no-translate`; bug?
> 
> ```
> $ trans -speak -no-translate hello   # not speak
> Voice output isn't available for English
> 
> $ trans -speak -b hello   # speak
> hello
> 
> $ trans -V
> Translate Shell       0.9.7-git:a3bba39
> …
> ```

This was a regression in 0.9.7; should be fixed in the develop branch (or the latest 0.9.7.1) already.
